Branch-SDK/src/io/branch/referral/RemoteInterface.java

```diff
@@ -132,33 +139,41 @@ public ServerResponse make_restful_get(String url, String tag, int timeout, bool
 		}
 		return null;
 	}
+    //endregion
 
+    //region POST requests
 	public ServerResponse make_restful_post(JSONObject body, String url, String tag, int timeout) {
-		return make_restful_post(body, url, tag, timeout, true, null);
+		return make_restful_post(body, url, tag, timeout, 0, true, null);
 	}
 	
 	public ServerResponse make_restful_post(JSONObject body, String url, String tag, int timeout, BranchLinkData linkData) {
-		return make_restful_post(body, url, tag, timeout, true, linkData);
+		return make_restful_post(body, url, tag, timeout, 0, true, linkData);
 	}
 
 	public ServerResponse make_restful_post(JSONObject body, String url, String tag, int timeout, boolean log) {
-		return make_restful_post(body, url, tag, timeout, log, null);
+		return make_restful_post(body, url, tag, timeout, 0, log, null);
 	}
 	
-	public ServerResponse make_restful_post(JSONObject body, String url, String tag, int timeout, boolean log, BranchLinkData linkData) {
+	public ServerResponse make_restful_post(JSONObject body, String url, String tag, int timeout,
```

Comment:
I think we had discussed this on the other PR, but should the method w/ the actual retryCount be public?